rpc_jobs/rpc_barbican.yml

```diff
@@ -0,0 +1,44 @@
+---     # Job definitions for Barbican integrated with Vault
+
+################################################################################################
+
+- project:
+    name: "rpc-barbican-pre-merge"
+    repo_name: "rpc-barbican"
+    repo_url: "https://github.com/rcbops/rpc-barbican"
+    branches:
+      - "rocky.*"
```

Comment:
There is no branch named ``rocky`` in the rpc-barbican repo. Will there be one in the future, or will this need to change to be ``master`` instead?

Reply:
I will create a rocky branch in the first commit.

Reply:
We have now a working rocky branch, please guide me to complete the setup